support/src/main/java/com/factual/cdh5docker/utils/ServiceLauncher.java

```diff
@@ -676,6 +670,7 @@ static void launchHBaseReadyStateThread(final LaunchConfig launchConfig, final C
       public void run() {
         Logger.getLogger("rg.apache.hadoop.hbase").setLevel(Level.ALL);
         try { 
+          waitForZookeeperToBeReady();
```

Comment:
I don't do it before launchHBaseReadyStateThread.   It used to be there, but I moved it here as to not block the starting of the HBase threads.